recipe/conda_build_config.yaml

```diff
@@ -414,6 +414,7 @@ readline:
   - 7.0
 r_base:
   - 3.4.1
+  - 3.5.1
```

Comment:
@CJ-Wright this would be the "big migration" (besides the compilers+Python 2.7) test you are looking for :smile:

Reply:
lol, I was just hoping to look into the future so we can plan for the migrations. I wasn't expecting someone to build a time machine and make the future now!

Reply:
:relieved: